BUILDING.md

```diff
@@ -21,89 +19,72 @@ Start by understanding the layout of the source code:
+ 
+ ##### Linux
+ * gcc 7
+ 
+ ##### OSX
+ * xcode v????
+ 
+#### Building USD 
 
-### OSX / Darwin
+We typically use the standard build steps from the USD repo instructions. The exact command line we are using are:
+* with python support: `python3 build_scripts/build_usd.py --build-monolithic --alembic --no-imaging --no-examples --no-tutorials ../artifacts/usd-v20.08`
+* no python support:   `python3 build_scripts/build_usd.py --build-monolithic --alembic --no-python --no-imaging --no-examples --no-tutorials ../artifacts/usd-v20.08_no_python`
 
-Similarly for OSX, an XCode project is provided in the src directory, called
-UsdCs.xcodeproj. The OSX build will produce a .bundle file, into which all
-dependent dylibs must be manually copied, as well as the USD plugInfo.json
-files. See the existing bundle as an example of the correct structure.
+**Unity developers**: USD binaries are built with each new release and pushed on Stevedore. See next section to download them automatically when building the bindings.
 
-The XCode project is setup to add the RPATH `@loaderpath/../Frameworks/` which
-is critical to enable the UsdCs.dylib to link against the external dylibs in
-the Frameworks directory. To verify the RPATH is set correcty, run
-`otool -l UsdCs.dylib` and search for `LC_RPATH`. If the RPATH is not set,
-it can be added to the library after compilation using the command
-```
-install_name_tool -add_rpath @loaderpath/../Frameworks/ UsdCs
-install_name_tool -add_rpath @loader_path/../../../ UsdCs
-```
+#### Building the C# bindings
 
-## Generating Bindings
+The bindings are made of two libraries:
+* UsdCs: the native USD C# bindings library (C++, SWIG generated)
+* USD.NET: the C# library (serialization API)
 
-C# The following instructions assume a Windows build environment, though
-the same process should work on OSX as well.
+The simplest way to build the bindings libraries is to use the provided python build script with the corresponding component (usdcs or usdnet)
 
-There are two main steps to code generation, the first is a sequence of
-Python scripts which generate type-specific SWIG shims (.i files). The
-second step is the SWIG code generator itself.
+Start by setting a USD_LOCATION environment variable pointing to a directory containing the 2 different flavors of USD (python and no puython)
 
- * The Python step has been tested with Ptyhon 2.7 and requires the USD
-   Python files to be importable via PYTHONPATH.
- * The SWIG step requires the SWIG 3.0.12 executable on your system PATH.
+In a terminal, running the following command from the root of the repository will build UsdCs by default:
 
-By setting USD_LOCATION_PYTHON to the root install directory of a USD
-python build, all scripts will work without setting the system PATH or
-PYTHONPATH. The variable USD_LOCATION should be set to a USD build without
-python, to minimize runtime dependencies of the final plugin.
+`python3 bin/build.py 20.08 $USD_LOCATION 2019.4`
```

Comment:
would be nice if you could use USD_LOCATION only if you can't download or are an advanced user:
Basically:
If !USD_LOCATION set
    If Can Download
       Download and set USD_LOCATION
    Warn user to set USD_LOCATION

use USD_LOCATION

:)

Reply:
Fixed in e38eb675
Now downloads in ./artifacts . No need for a USD_LOCATION var anymore